### PR TITLE
Update html5ever to 0.10.1

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14164)

### CODE REVIEW

File: components/script/dom/htmlscriptelement.rs [hunk 1]
```diff
@@ -275,10 +274,10 @@ fn fetch_a_classic_script(script: &HTMLScriptElement,
 
 impl HTMLScriptElement {
     /// https://html.spec.whatwg.org/multipage/#prepare-a-script
-    pub fn prepare(&self) -> NextParserState {
+    pub fn prepare(&self) -> bool {
```
Comment (frewsxcv) [inline components/script/dom/htmlscriptelement.rs]: What does this returned `bool` correspond to?

Comment (nox) [inline components/script/dom/htmlscriptelement.rs]: Oh right. Added a comment.

Comment (frewsxcv) [inline components/script/dom/htmlscriptelement.rs]: > Added a comment.

❤️

**Pull request merged**